cmd/pinniped-proxy/Cargo.toml

```diff
@@ -17,4 +17,5 @@ log = "0.4"
 pretty_env_logger = "0.4"
 structopt = "0.3"
 # Update to 0.3 once apparent issue with the nascent 0.3.5 is fixed.
-tokio = { version = "0.2", features = ["full"] }
\ No newline at end of file
+tokio = { version = "0.2", features = ["full"] }
+url = "2.1"
```

Comment:
you are not adding a blank line at the end for any reason?

Reply:
Yeah, I was working on a different computer and didn't have my editor there setup to do it automatically. I did notice and fix it in the following one, I think.